Vector.C

```diff
@@ -532,4 +532,60 @@ Vector::local_read(const std::string& base_file_name, int rank)
     database.close();
 }
 
+int getCenterPoint(std::vector<Vector*> points,
```

Comment:
Can this function replace the similar code in `GreedyParameterPointSampler::getNextParameterPoint()`?

Reply:
Good catch